vm/api/src/errors/DumpStateError.ts

```diff
@@ -0,0 +1,8 @@
+import type { InvalidRequestError } from './InvalidRequestError.js'
+import type { UnexpectedError } from './UnexpectedError.js'
+
+/**
+ * Error Returned by dump state procedure
+ * TODO : Give example call
+ */
```

Comment:
```suggestion
// TODO : Give example call
/**
 * Error Returned by dump state procedure
 */
```
Keep todos out of jsdoc since end users see jsdoc